modules/bookmanager.js

```diff
@@ -0,0 +1,67 @@
+export default class BookManager {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+    this.booksDiv = document.getElementById('bookContainer');
+    this.addBookForm = document.getElementById('addBookForm');
+    this.titleInput = document.getElementById('titleInput');
+    this.authorInput = document.getElementById('authorInput');
+
+    this.displayBooks();
+    this.addEventListeners();
+  }
+
+  displayBooks() {
+    this.booksDiv.innerHTML = '';
+
+    if (this.books.length === 0) {
+      this.booksDiv.textContent = 'No books in the collection.';
+      return;
+    }
+
+    const removeBook = (index) => {
+      this.books.splice(index, 1);
+      localStorage.setItem('books', JSON.stringify(this.books));
+      this.displayBooks();
+    };
+
+    this.books.forEach((book, index) => {
+      this.booksDiv.innerHTML += `<div class="book">
+              <span class="book-title">${book.title}</span>
+              <span class="book-author">by ${book.author}</span>
+              <button class="remove-button" type="button" data-index="${index}">Remove</button>
+            </div>`;
+    });
+
+    const removeButtons = document.getElementsByClassName('remove-button');
+    Array.from(removeButtons).forEach((button, index) => {
+      button.addEventListener('click', () => {
+        removeBook(index);
+      });
+    });
+  }
+
+  addBook(title, author) {
+    const book = {
+      title,
+      author,
+    };
+
+    this.books.push(book);
+    localStorage.setItem('books', JSON.stringify(this.books));
+    this.displayBooks();
+  }
+
+  addEventListeners() {
+    this.addBookForm.addEventListener('submit', (event) => {
+      event.preventDefault();
+      this.addBook(this.titleInput.value, this.authorInput.value);
+      this.titleInput.value = '';
+      this.authorInput.value = '';
+    });
```

Comment:
- [ ] As part of the project requirements kindly make sure to modularize your code as there's still room for here, for example, you can place the different operations like add book, remove book, and display book in their own separate modules (or a single module called `operations`) and implement them as functions
- As an example the add book can be replaced as below
```js
import { displayBooks } from './displayBooks.js';

export function addBook(title, author, books, booksDiv, removeBook) {
  const book = {
    title,
    author,
  };

  books.push(book);
  localStorage.setItem('books', JSON.stringify(books));
  displayBooks(books, booksDiv, removeBook);
}
```
- Please keep in mind that this is just for demonstration it might or might not work. 😅